superflore/generators/bitbake/ros_meta.py

```diff
@@ -13,7 +13,8 @@
 # limitations under the License.
 
 
-from superflore import repo_instance
+# TODO(allenh1): remove inheritance like the RosOverlay class.
+from superflore import RepoInstance as repo_instance
```

Comment:
This is a temporary hacky-fix to avoid breaking things in the OpenEmbedded world. This will be updated when I catch OpenEmbedded up.